src/types.rs

```diff
@@ -15,6 +17,12 @@ pub struct Coordinate<T>
     pub y: T,
 }
 
+impl<T: Float> From<(T, T)> for Coordinate<T> {
```

Comment:
Remember, rust-geo is still 0.x.y, so according to semver you can change things however you want. :)